bin/cli.js

```diff
@@ -190,11 +190,13 @@ function invoke(env) {
     });
 
   commander
-    .command('migrate:up')
-    .description('        Run the next migration that has not yet been run.')
-    .action(() => {
+    .command('migrate:up [<name>]')
+    .description(
+      '        Run the next or the defined migration that has not yet been run.'
```

Comment:
`specified` is likely to be clearer than `defined`

Reply:
done